src/main/java/mil/dds/anet/resources/PositionResource.java

```diff
@@ -229,4 +229,70 @@ public Integer deletePosition(@GraphQLRootContext Map<String, Object> context,
     return dao.delete(positionUuid);
   }
 
+  @GraphQLMutation(name = "mergePositions")
+  public Position mergePositions(@GraphQLRootContext Map<String, Object> context,
+      @GraphQLArgument(name = "loserUuid") String loserUuid,
+      @GraphQLArgument(name = "winnerPosition") Position winnerPosition) {
+    final Person user = DaoUtils.getUserFromContext(context);
+    final Position existingPos = dao.getByUuid(loserUuid);
+
+    // Check that given two position can be merged
+    arePositionsMergeable(existingPos, winnerPosition);
+    assertCanUpdatePosition(user, winnerPosition);
```

Comment:
I would first do the assert, then check the positions for mergeability/validity.